src/bindings/c/include/openvino/c/ov_shape.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief This is a header file for ov_shape C API
+ *
+ * @file ov_shape.h
+ */
+
+#pragma once
+
+#include "openvino/c/ov_common.h"
+
+/**
+ * @struct ov_shape_t
+ * @brief Reprents a static shape.
+ */
+typedef struct {
+    int64_t rank;
+    int64_t* dims;
+} ov_shape_t;
+
+/**
+ * @brief Init a shape object, allocate space for its dimensions.
+ * @ingroup shape
+ * @param rank The rank value for this object, it should be more than 0(>0)
+ * @param ov_status_e a status code.
+ */
+OPENVINO_C_API(ov_status_e) ov_shape_init(ov_shape_t* shape, int64_t rank);
```

Comment:
how to create shape from array? E.g. { 1, 2, 3, 4 } ?

Reply:
ov_shape_t shape;
    ov_shape_init(shape, 4);
    shape->dims[0] = 1;
    shape->dims[1] = 2;
    shape->dims[2] = 3;
    shape->dims[3] = 4;
    ...
    ov_shape_deinit(&shape);

    
    Actually, I have a long discussion with Ilya.C about ov_shape, in my proposal, we can define a simple fixed array to represent ov_shape and didn't need such complicate steps to create ov_shape, but Ilya.C disagree such solution, Maybe you can check the discussion history.

Reply:
sure, add below api:

ov_shape_create(ov_shape_t* shape, int64_t rank, int64_t* dims);